image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -117,6 +117,9 @@ spec:
           mountPropagation: HostToContainer
         {{- end }}
         [<- end >]
+        - mountPath: "/run/collector_runtime_config"
```

Comment:
Honestly, configuration should live in `/etc` IMO, `/run` is for things that get created by a process at runtime (sockets, pid files, etc...). I also agree it should be more generic so it can be extended, in my  mind, collector configuration should live under `/etc/stackrox/collector.yml`, but I guess that is one thing we could try to change in the future, just make sure to add an environment variable to collector so we can point it to a different file if needed.

Reply:
I have set the `mountPath` to `/etc/stackrox`.